ui/src/app/base/components/FormCardButtons/FormCardButtons.js

```diff
@@ -1,15 +1,19 @@
 import { ActionButton, Button } from "@canonical/react-components";
-import { useRouter } from "app/base/hooks";
 import PropTypes from "prop-types";
 import React from "react";
 
+import { useRouter } from "app/base/hooks";
+
 export const FormCardButtons = ({
   actionDisabled,
   actionLabel,
   actionLoading,
-  actionSuccess
+  actionSuccess,
+  handleSecondarySubmit,
```

Comment:
I feel like it is a bit weird that we have two handlers, with quite different names, `actionSuccess` and `handleSecondarySubmit`.

Reply:
I agree, but then again this whole pattern is a bit weird. `handleSecondarySubmit` is actually the only submit handler that this component knows about. The normal submit is handled by the parent form element (via the `type="submit"` button) and `actionSuccess` isn't actually a submit handler, it's just to show the success icon in the ActionButton. I'll try and come up with some clearer names.

Reply:
Ah I see, that's quite fine in that case, sorry for the confusion. Thanks @Caleb-Ellis